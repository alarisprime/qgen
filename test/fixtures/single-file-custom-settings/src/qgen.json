{
	"directory": "./my-templates",
	"dest": "./pages",
	"templates": {
		"blog.md": {
			"title": "A Fresh Title",
			"slug": "a-fresh-title"
		}
	}
}
