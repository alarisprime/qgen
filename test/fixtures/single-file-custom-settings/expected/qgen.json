{
	"directory": "./my-templates",
	"templates": {
		"blog.md": {
			"title": "A Fresh Title",
			"slug": "a-fresh-title",
			"dest": "./pages"
		}
	}
}
